_data/simple-icons.json

```
@@ -314,6 +314,11 @@
             "source": "https://www.adyen.com/press-and-media/presskit",
             "guidelines": "https://www.adyen.com/press-and-media/presskit"
         },
+        {
+            "title": "Aegis Authenticator",
+            "hex": "005E9D",
+            "source": "https://alexbakker.me/u/pn8frf828n.svg"
```

```suggestion
            "source": "https://getaegis.app/aegis.svg"
```
Thank you again @michaelschattgen!